app/src/components/email.js

```diff
@@ -110,23 +136,26 @@ const email = {
    *  @returns {string} The url of the generated Ethereal email
    */
   sendMailEthereal: async message => {
-    // Generate test SMTP service account from ethereal.email
-    // Only needed if you don't have a real mail account for testing
-    const testAccount = await nodemailer.createTestAccount();
-    log.debug(utils.prettyStringify(testAccount));
-
-    // Create reusable transporter object using the default SMTP transport
-    const transporter = nodemailer.createTransport({
-      host: testAccount.smtp.host,
-      port: testAccount.smtp.port,
-      auth: {
-        user: testAccount.user,
-        pass: testAccount.pass
-      }
-    });
+    if (!etherealTransporter) {
```

Comment:
I think it's time to remove ethereal, as we can't support this in OpenShift.
Put it on the backlog if we need it, we put it back in.  Right now it's noise and clutter.  If anything, the transport could be passed in for unit testing, but not part of the API.  
Let's discuss and see if we should create backlog items.

Reply:
Had a quick chat about this with @loneil and believes it should remain in the codebase as is. While it may not work on the main case (our Openshift), it isn't to say it won't work on other environments (Kube or other OC with less network restrictions). It doesn't actually hurt to leave it in and it's already unit tested.